### PR TITLE
chore(evm) - Delete deprecated store migrations

### PR DESCRIPTION
**Pull request by Vvaradinov**
This PR deletes old migrations which rely on the now deprecated `x/params` module. 

Closes: #XXX

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: x/evm/module.go [hunk 1]
```diff
@@ -123,15 +123,6 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
 	types.RegisterMsgServer(cfg.MsgServer(), am.keeper)
 	types.RegisterQueryServer(cfg.QueryServer(), am.keeper)
```
Comment (fedekunze) [inline x/evm/module.go:125]:
```suggestion
```

**Review by fedekunze — APPROVED** [review]
ACK. Pending linter fix

**Comment by yihuang** [comment]
How do you do migration in upgrade if you deleted them?

**Comment by Vvaradinov** [comment]
> How do you do migration in upgrade if you deleted them?

@yihuang I think these migrations will not affect upgrade handlers as another PR https://github.com/evmos/ethermint/pull/1472 will migrate to the new Cosmos SDK params scheme and probably delete / deactivate the `x/params` module in the upgrade.

**Comment by yihuang** [comment]
> > How do you do migration in upgrade if you deleted them?
> 
> @yihuang I think these migrations will not affect upgrade handlers as another PR #1472 will migrate to the new Cosmos SDK params scheme and probably delete / deactivate the `x/params` module in the upgrade.

I see, the consensus version is bumped to 4, I guess it's fine.

**Pull request merged**